app/invocation/invocation_targets_card.tsx

```diff
@@ -68,7 +68,10 @@ export default class TargetsCardComponent extends React.Component<Props, State>
               {events
                 .slice(0, (this.props.pageSize && this.state.numPages * this.props.pageSize) || undefined)
                 .map((target) => (
-                  <Link className="target-row" href={`?target=${encodeURIComponent(target.id.targetCompleted.label)}`}>
+                  <Link
+                    className="target-row"
+                    draggable={false}
```

Comment:
is `draggable={false}` needed? on Linux at least, it looks like `user-select: text;` does the trick on its own

Reply:
You're right, removed